src/core/Akka/IO/SocketEventArgsPool.cs

```diff
@@ -42,12 +40,9 @@ public PreallocatedSocketEventAgrsPool(int initSize, EventHandler<SocketAsyncEve
 
         public SocketAsyncEventArgs Acquire(IActorRef actor)
         {
-            SocketAsyncEventArgs e;
-            if (!_pool.TryPop(out e))
-            {
+            if (!_pool.TryPop(out var e))
```

Comment:
Should we just get rid of this class altogether, and the HOCON that goes with it?

Reply:
We wanted to keep pooling for UDP, right? That's why there is still some configuration for this in HOCON, and this class itself. 
I did not look into UDP implementation yet, so not sure we should touch it (and if it will improve anything).